resource/logging_macros.h.em

```diff
@@ -24,6 +24,13 @@
 #include <stdio.h>
 #include <stdlib.h>
 
+// This is to avoid compilation warnings in C++ with '-Wold-style-cast'.
+#ifdef __cplusplus
+  #define CAST_DURATION(x) (static_cast < rcutils_duration_value_t > (x))
```

Comment:
> Uncrustifying that file adds the spaces, so I guess that's the expected style?

Oh, right.  I think this is because we are adding a C++ construct into a C header file.  Uncrustify thinks that this is the "greater than"/"less than" operators in C, and so wants spaces around it.

All right, in that case, let's leave this as-is.  I'm a little worried that this is going to cause other things to be unhappy, but we'll see what CI has to say about it.